Formula/astrometry-net.rb

```diff
@@ -0,0 +1,46 @@
+class AstrometryNet < Formula
+  desc "Automatic identification of astronomical images"
+  homepage "http://astrometry.net/"
+  url "https://github.com/dstndstn/astrometry.net/releases/download/0.76/astrometry.net-0.76.tar.gz"
+  sha256 "244355ef9716a9e062eb19e8547e8b50434c7df52a8b96f9d0a254bc646d1f0d"
+
+  depends_on "pkg-config" => :build
+  depends_on "swig" => :build
+  depends_on "cairo"
+  depends_on "cfitsio"
+  depends_on "gsl"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "netpbm"
+  depends_on "numpy"
+  depends_on "python"
+  depends_on "wcslib"
+
+  def install
+    ENV["INSTALL_DIR"] = prefix
+    ENV["NETPBM_INC"] = "-I#{Formula["netpbm"].opt_include}/netpbm"
+    ENV["NETPBM_LIB"] = "-L#{Formula["netpbm"].opt_lib} -lnetpbm"
+    ENV["SYSTEM_GSL"] = "yes"
+    ENV["PYTHON"] = "python3"
+    ENV["PYTHON_SCRIPT"] = "/usr/bin/env python3"
+    xy = Language::Python.major_minor_version "python3"
+    ENV["PY_BASE_INSTALL_DIR"] = "#{lib}/python#{xy}/site-packages/astrometry"
+
+    system "make"
+    system "make", "py"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/build-astrometry-index", "-d", "3", "-o", "index-9918.fits", "-P", "18", "-S", "mag",
```

Comment:
Great! Can you also make sure that these lines doesn't surpass the 80 character limit ? Thanks!

Reply:
sure, done :)